crates/primitives/src/transaction/signature.rs

```diff
@@ -100,7 +100,26 @@ impl Signature {
         secp256k1::recover(&sig, hash.as_fixed_bytes()).ok()
     }
 }
-
+impl From<&Signature> for [u8; 65] {
+    fn from(src: &Signature) -> [u8; 65] {
+        let mut sig = [0u8; 65];
+        let r_bytes: [u8; 32] = src.r.to_be_bytes();
+        let s_bytes: [u8; 32] = src.s.to_be_bytes();
+        sig[..32].copy_from_slice(&r_bytes);
+        sig[32..64].copy_from_slice(&s_bytes);
```

Comment:
nit: please no additional step

pass the function result directly into copy_from_slice